theme/static/css/hydroshare_core.css

```diff
@@ -609,37 +639,37 @@ footer .headline h3{
     margin: 0;
 	padding: 0;
 }
+
 .social li {
     float: left;
     display: block;
     list-style-type: none;
     width: 28px;
     height: 28px;
-    background: #eee;
     margin: 0 5px 5px 0;
     text-align: center;
     line-height: 28px;
 }
+
 .social li:hover {
     background: none;
 }
-.social li, .social li:hover {
-    transition: background 0.1s;
-    -webkit-transition: background 0.1s;
-    -moz-transition: background 0.1s;
-}
+
 .social li a {
-    color: #3E7A8C;
+    color: #FFF;
 
 }
+
 .social li:hover a {
```

Comment:
This selector is the same as the one right above it, so these properties can be combined under one or the other.